pints/__init__.py

```diff
@@ -140,6 +140,7 @@ def version(formatted=False):
     SumOfErrors,
```

Comment:
This one's also wrong :-D

Reply:
Oh! OK, you just answered my question above. I will move it too then.

Reply:
Is it just for the docs/__init__.py only? Or I need to move it for the actual _error_measures.py functions as well...?!

Reply:
docs, imports, and actual code!

Reply:
We've done it in _most_, but not all places, so you might see some other instances where it's wrong